mac-bundle.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+
+app=NooDS.app
+contents=$app/Contents
+
+if [[ ! -f noods ]]; then
+	echo 'Error: NooDS binary was not found.'
+	echo 'Please run `gmake` to compile NooDS before bundling.'
```

Comment:
Shouldn't it just be `make`? Also, I realized mac-bundle.sh and Info.plist use tabs instead of spaces; not sure if that matters or if it should match the code which uses spaces.